### DISCUSSION TITLE
Vimar 14592.0 Power on behavior errors in HA

### DISCUSSION THREAD

**Discussion by avvalex** [Koenkk/zigbee2mqtt#20854] [Devices]
Hi, i have several switches Vimar 14592.0, and in HA i have thousands of these errors:

Invalid option for select.garden_power_on_behavior: '' (valid options: ['off', 'on', 'toggle', 'previous'])

Is it something that can be fixed? If i try to set Power on behavior in z2m It says that the device doesn't supporto it....

Thanks!!

**Comment by Koenkk** [comment]
Disabled it since the device does not support it!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

**Comment by BB1982** [comment]
Hi All!
I'm writing here just to ask whether the Vimar 14592 supports direct bind. My idea would be to control the load switched by one 14592 using another 14592 without using a hub (besides initial config of course!)
